From 91edfa920c13a1d542923f211228d4af3dcc4dde Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Tue, 22 Dec 2020 10:50:54 +0700
Subject: [PATCH 356/357] arm64: dts: renesas: cat874-common: Add reg_1p8v and
 reg_3p3v

This patch supports reg_1p8v and reg_3p3v to cat874-RZG2 boards.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm64/boot/dts/renesas/cat874-common.dtsi | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/cat874-common.dtsi b/arch/arm64/boot/dts/renesas/cat874-common.dtsi
index a347030..c7c4b0e 100644
--- a/arch/arm64/boot/dts/renesas/cat874-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/cat874-common.dtsi
@@ -84,6 +84,24 @@
 		compatible = "renesas,vspm_if";
 	};
 
+	reg_1p8v: regulator0 {
+		compatible = "regulator-fixed";
+		regulator-name = "fixed-1.8V";
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
+		regulator-boot-on;
+		regulator-always-on;
+	};
+
+	reg_3p3v: regulator1 {
+		compatible = "regulator-fixed";
+		regulator-name = "fixed-3.3V";
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+		regulator-boot-on;
+		regulator-always-on;
+	};
+
 	reg_12p0v: regulator-12p0v {
 		compatible = "regulator-fixed";
 		regulator-name = "D12.0V";
-- 
2.7.4

